readme.md

```diff
@@ -27,11 +29,12 @@ Make all the changes smooth on hover (during 300ms):
 
 > Here are the [Layout Tasks Instructions](https://mate-academy.github.io/layout_task-guideline)
 
+
 ## Checklist
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
-- [DEMO LINK](https://<your_account>.github.io/layout_catalog/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_catalog/report/html_report/)
+- [DEMO LINK](https://Razsinxron.github.io/layout_catalog/)
+- [TEST REPORT LINK](https://Razsinxron.github.io/layout_catalog/report/html_report/)
```

Comment:
Add these links to the PR description 
<img width="940" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/64750958/fc0e9746-c55c-47b0-8542-df3d8ec57726">